demos/microservices/service_c.py

```diff
@@ -23,7 +23,7 @@
     app = web.Application()
     app.router.add_get('/api/v1/data', handler)
 
-    zipkin_address = 'http://127.0.0.1:9411'
+    zipkin_address = 'http://127.0.0.1:9411/api/v2/spans'
```

Comment:
should this property be renamed as url or endpoint?

Reply:
Good point, for some reason I did not think about this.